pagespeed/envoy/envoy_cluster_manager.cc

```diff
@@ -107,13 +109,38 @@ void EnvoyClusterManager::initClusterManager() {
       dispatcher_->createDnsResolver({}), *ssl_context_manager_, *dispatcher_, *local_info_,
       secret_manager_, validation_context_, *api_, http_context_, *access_log_manager_,
       *singleton_manager_);
+}
 
-  Envoy::MessageUtil::loadFromFile(
-      "pagespeed/envoy/cluster.yaml",
-      bootstrap, Envoy::ProtobufMessage::getStrictValidationVisitor(), *api_);
-
-  cluster_manager_ = cluster_manager_factory_->clusterManagerFromProto(bootstrap);
+Envoy::Upstream::ClusterManager&
+EnvoyClusterManager::getClusterManager(const GoogleString str_url_) {
+  UriImpl uri(str_url_);
+  try {
+    uri.resolve(*dispatcher_, Envoy::Network::DnsLookupFamily::Auto);
```

Comment:
We probably don't want to resolve at this point in time?

Reply:
removed in latest code 
code pushed in https://github.com/apache/incubator-pagespeed-mod/pull/1969/commits/4ab87afdc150b403bcef176c8f4934e8f4ba18d1